compiler/rustc_sanitizers/src/cfi/typeid/itanium_cxx_abi/encode.rs

```diff
@@ -99,6 +105,7 @@ fn encode_args<'tcx>(
 fn encode_const<'tcx>(
     tcx: TyCtxt<'tcx>,
     c: Const<'tcx>,
+    ct_ty: Ty<'tcx>,
```

Comment:
Could Const and Ty be rebundled in a struct (e.g., ConstAndTy)  just for this file for consistency a with other methods?

Reply:
Realistically this ty ought to be unnecessary, theres no reason cfi should be having to deal with `ConstKind::Param`. I want to land that separately after this PR incase it turns out to cause ICE